src/static/css/2019.css

```diff
@@ -972,6 +972,11 @@ p.copyright a {
   display: none;
 }
 
+div.js-show,
+#menu.js-show {
```

Comment:
Are the element/ID important for specificity or can we use the plain `.js-show` selector? I'd also suggest combining with the `.hidden` rule above:

```css
.hidden,
.js-show {
  display: none;
}

Reply:
They were needed for specificity - but changed the code a bit since I wrote this and no longer needed. Hurray!

Good call in merging too.

Fixed.